src/util/run-plugin-promise.js

```diff
@@ -21,16 +21,20 @@ module.exports = function runPluginFunction(params, functionName) {
         } else {
           pluginName = plugin
         }
-
-        var tmp = require(pluginName)
-        var has = tmp.hasOwnProperty(functionName)
-        var fn = tmp[functionName]
+        if (pluginName.split('').includes('/')) {
```

Comment:
We should probably also check for a backslash for other platforms

Reply:
This bit is there specifically for NPM scoping (eg `@architect/functions`) so we can rely on it being a forward slash, but per your point above it does need to be interpolated to backslash on Windows when running a local absolute path based `require`.

Reply:
Ah got it. Thanks for explaining.